canister/src/types.rs

```diff
@@ -445,7 +465,42 @@ impl Storable for (Height, OutPoint) {
 pub type BlockBlob = Vec<u8>;
 
 // A blob representing a block header in the standard bitcoin format.
-pub type BlockHeaderBlob = Vec<u8>;
+#[derive(CandidType, PartialEq, Clone, Debug, Eq, Serialize, Deserialize, Hash)]
+pub struct BlockHeaderBlob(Vec<u8>);
+
+impl StableStructuresStorable for BlockHeaderBlob {
+    fn to_bytes(&self) -> std::borrow::Cow<[u8]> {
+        self.0.to_bytes()
+    }
+
+    fn from_bytes(bytes: Vec<u8>) -> Self {
+        Self::from(bytes)
+    }
+}
+
+impl BoundedStorable for BlockHeaderBlob {
+    fn max_size() -> u32 {
+        80
```

Comment:
Might be worth explaining where we get this "magic" number from. Having a constant for it would be nice as well.